routes/api/events.js

```diff
@@ -59,6 +61,20 @@ router.get('/inactive', async (req,res)=>{
 
 });
 
+//endpoint para obtener eventos por id
+router.get('/:eventId',isAdmin, async (req,res)=>{
+    if (isNaN(req.params.eventId)) {
```

Comment:
```suggestion
    if (Number.isNaN(req.params.eventId)) {
```
Mejor usa Number.isNaN(), no isNaN()